idaes/commands/extensions.py

```diff
@@ -152,3 +181,41 @@ def get_extensions(
             print_extensions_version(library_only)
     else:
         click.echo("\n* You must provide a download URL for IDAES binary files.")
+
+
+@cb.command(name="hash-extensions", help="Calculate release hashes")
+@click.option(
+    "--release",
+    help="Optional, specify an official binary release to download",
+    default=None,
+    required=True)
+@click.option(
+    "--path",
+    help="Directory of release files",
+    default=None,
+    required=True)
+def hash_extensions(release, path):
+    hfile = f"sha256sum_{release}.txt"
+    if path is not None:
+        hfile = os.path.join(path, hfile)
+
+    def _w(fp, pack, plat):
```

Comment:
maybe give this a longer name to make it easier to follow this code, `_write_hashes`?